bazel_tools/scala.bzl

```diff
@@ -201,6 +201,121 @@ def _create_scala_source_jar(**kwargs):
             srcs = kwargs["srcs"],
         )
 
+def _scaladoc_jar_impl(ctx):
+    srcFiles = [
+        src.path
+        for src in ctx.files.srcs
+    ]
+    pluginPaths = []
+    for p in ctx.attr.plugins:
+        if hasattr(p, "path"):
```

Comment:
IIRC you mentioned this part is based on a similar piece of code in some other rule set? Could you add a comment pointing there for reference?

Reply:
Done